pinot-plugins/pinot-input-format/pinot-avro-base/src/main/java/org/apache/pinot/plugin/inputformat/avro/AvroRecordReaderConfig.java

```diff
@@ -0,0 +1,36 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.pinot.plugin.inputformat.avro;
+
+import org.apache.pinot.spi.data.readers.RecordReaderConfig;
+
+/**
+ * Config for {@link AvroRecordReader}
+ */
+public class AvroRecordReaderConfig implements RecordReaderConfig {
+  private boolean _enableLogicalTypes;
+
+  public boolean isEnableLogicalTypes() {
```

Comment:
For ingestion jobs this properties can be configured via recordsReaderSpec. However I can't see how these property are configured for a realtime table - perhaps via ingestionSpec in the table definition? I was looking through the code but I couldn't figure it out. I believe this is not possible at the moment

Reply:
I think I've found where the properties can be read from. Stream config has decoder properties (it looks like these are available in table config of realtime tables) which are propagated to the Avro decoder classes, it looks like I can take advantage of that to populate the logical type flag. I'll add a commit to the PR and let me know if you see anything missing @walterddr

Reply:
hmm. not sure which one are you referring to. there's a `RecordReaderConfig` base class which provides configuration properties to `RecordReader`s (see `RecordReaderFactory` class). and it is apparently passed in via a `readerConfigFile` path (see. `CreateSegmentCommand` entrypoint)